src/dynamic-side-content.scss

```diff
@@ -0,0 +1,101 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-dynamic-side;
+
+.#{$block} {
+  $main: #{$block}__main;
+  $side: #{$block}__side;
+
+  $main-width-screen-s: 100%;
+  $main-width-screen-m: 66.666%;
+  $main-width-screen-xl: 75%;
+
+  @include fd-reset();
+  @include fd-flex();
+
+  width: 100%;
+
+  .#{$main} {
+    @include fd-reset();
+
+    @include fd-screen(s) {
```

Comment:
Could you change it to `--sm/--md/--xl` modifiers? We are trying to get rid of those breakpoints related mixins, I know that It is defined in specs, but you can show it on examples, by wrapping dynamic side content inside `div` with some `max-width`

Reply:
Since this layout relies on screen width we need to render it properly we have to add additional class as e.g. .#{$block}--xl to have it not broken.

Reply:
@JKMarkowski we need to have some default look (when we do not specify --size modifier), so what variant should be default one `sm`/`md`/`xl` ?

Reply:
I removed `fd-screen` usage and made `xl` behaviour as the default one

Reply:
Also added example for `responsivness`